### PR TITLE
Render fully qualified type names.

### LINKED ISSUE #30: Render fully-qualified type names.
Especially with the introduction of Go modules, it's common to have two types with the same name and package name.

Showing fully qualified type names would satisfy Dapper's design goal of being unambiguous. (at the cost of being more verbose).

### PR DESCRIPTION
**Pull request by jmalloc**
This PR changes dapper to render fully-qualified package paths (fixes #30) in type names by default, and adds a new configuration option to disable this behaviour.

### CODE REVIEW

**Comment by danilvpetrov** [comment]
@jmalloc, @koden-km: I've added the test for `Printer.Config.Config.OmitPackagePaths` option, which is a copy-and-paste of `Example_Printer` test with slight change. Can you please take a quick look?

File: printer_test.go [hunk 1]
```diff
@@ -51,6 +96,7 @@ func ExamplePrinter() {
 	//         }
 	//     }
 	// }
+
```
Comment (koden-km) [inline printer_test.go]: @danilvpetrov It looks good to me, except for this extra new line.

Comment (danilvpetrov) [reply printer_test.go]: Ok, I removed that.

**Comment by danilvpetrov** [comment]
Ok, guys I am merging this PR to proceed with #26. Thank you.

**Pull request merged**